main/main.go

```diff
@@ -223,6 +223,9 @@ func initIBU(ctx context.Context, c client.Client, log *logr.Logger) error {
 		return err
 	}
 
+	// Strip the ResourceVersion, otherwise the restore fails
+	ibu.SetResourceVersion("")
```

Comment:
This is actually not needed. I forgot to clean it up from the saving function. It was needed when I had the restore in a script where the curl commands for post and patch use the same source file. With api calls, we get the right revision in the structure after the create succeeds.

Reply:
I'm pretty sure I hit the same issues with the seedgen CR. I'd rather leave it here, and we can remove it in a separate PR if it's truly not needed.